Patch104pZH/GameFilesEdited/Data/INI/Armor.ini

```
@@ -479,11 +483,12 @@ End
 ;****************************************************
 ;***IF YOU CHANGE THESE, CHANGE ComancheArmor TOO!***
 ;****************************************************
+; Patch104p @balance Change EXPLOSION from 90%
```

Is this comment correct? Shouldn't it be from 110%?
```suggestion
; Patch104p @balance Change EXPLOSION from 110%
```

---

Fixed.